.storybook/stories/checkbox-toggle/checkbox-toggle-container.stories.ts

```diff
@@ -5,37 +5,21 @@
  */
 
 import { ClrCheckboxContainer, ClrCheckboxModule } from '@clr/angular';
-import { Parameters } from '@storybook/addons';
-import { Story } from '@storybook/angular';
-
-import { setupStorybook } from '../../helpers/setup-storybook.helpers';
+import { moduleMetadata, Story, StoryObj } from '@storybook/angular';
+import { CommonModules } from 'helpers/common';
```

Comment:
```suggestion
import { CommonModules } from '../../helpers/common';
```

This looks like a package import without the `../` parts.